headerCheck/factory/coreComponents.go

```diff
@@ -20,10 +20,11 @@ const (
 )
 
 type coreComponents struct {
-	Marshaller      marshal.Marshalizer
-	Hasher          hashing.Hasher
-	Rater           nodesCoordinator.ChanceComputer
-	PubKeyConverter core.PubkeyConverter
+	Marshaller          marshal.Marshalizer
+	Hasher              hashing.Hasher
+	Rater               nodesCoordinator.ChanceComputer
+	PubKeyConverter     core.PubkeyConverter
+	EnableEpochsHandler config.EnableEpochs
```

Comment:
this was added but it's not actually being set

Reply:
deleted